python/graphscope/deploy/kubernetes/resource_builder.py

```diff
@@ -799,6 +804,7 @@ def add_engine_container(self, name, image, cpu, mem, preemptive, **kwargs):
                 }
             )
         )
+        super.add_annotation("kubectl.kubernetes.io/default-container", name)
```

Comment:
This should be `super().add_annotation(`

Reply:
Sry, fixed~